notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -240,24 +250,13 @@ public Page createNote(Wiki noteBook, String parentNoteName, Page note, Identity
         if (properties != null) {
           DraftPage draftPage = getDraftNoteById(String.valueOf(properties.getNoteId()), userIdentity.getUserId());
           if (draftPage != null) {
-            NoteFeaturedImage featuredImage = properties.getFeaturedImage();
-            if (featuredImage != null && featuredImage.getId() != null && featuredImage.getId() > 0) {
-              moveOrCopyNotePageProperties(draftPage,
-                                           createdPage,
-                                           draftPage.getLang(),
-                                           note.getLang(),
-                                           NOTE_METADATA_DRAFT_PAGE_OBJECT_TYPE,
-                                           NOTE_METADATA_PAGE_OBJECT_TYPE,
-                                           userIdentity.getUserId(),
-                                           true);
-            } else {
-              properties.setNoteId(Long.parseLong(createdPage.getId()));
-              properties.setDraft(false);
-              properties = saveNoteMetadata(properties,
-                                            note.getLang(),
-                                            Long.valueOf(identityManager.getOrCreateUserIdentity(userIdentity.getUserId())
-                                                                        .getId()));
-            }
+            properties.setNoteId(Long.parseLong(createdPage.getId()));
+            properties.setDraft(false);
+            properties =
+                       saveNoteMetadata(properties,
```

Comment:
do we have the case the the featured image is already created by the last draft ?

Reply:
already handled in the `saveMetadata` function